_config/security.yml

```diff
@@ -1,4 +1,33 @@
-SilverStripe\Security\MemberLoginForm:
+SilverStripe\Security\MemberAuthenticator\LoginForm:
   required_fields:
     - Email
-    - Password
\ No newline at end of file
+    - Password
+
+SilverStripe\Security\Security:
+  authenticators:
+    default: SilverStripe\Security\MemberAuthenticator\Authenticator
+    cms: SilverStripe\Security\MemberAuthenticator\CMSAuthenticator
```

Comment:
No, it wouldn't. you can key your array just as you have above, and you can override it with a like yml config block. All that you are changing is moving from injecting a class name to a class instance, and the config is moved up one level to the Injector top level key.